modules/ROOT/partials/configuration/exportpdf.adoc

```diff
@@ -0,0 +1,51 @@
+[[exportpdf-service-url]]
+== `exportpdf_service_url`
+
+By default, the Export to PDF feature uses the {productname} Cloud Services HTML to PDF converter service to generate the `document-name.pdf` files. You can use this option to provide the URL to an on-premises converter. https://www.tiny.cloud/contact/[Contact us] if you require this feature.
+
+*Type:* `+String+`
+
+*Default value:* `true`
+
+=== Example: using `exportpdf_service_url`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'exportpdf',
+  toolbar: 'exportpdf',
+  exportpdf_service_url: '<service_url>'
+});
+----
+
+[[exportpdf-converter-options]]
+== `exportpdf_converter_options`
+
+This option allows the editor to download a PDF document containing its content, along with the specified header defined in the `exportpdf_converter_options`.
+
+*Type:* `+Object+`
+
+*Default value:* `{}`
+
+=== Example: using `exportpdf_converter_options`
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'exportpdf',
+  toolbar: 'exportpdf',
+  exportpdf_service_url: '<service_url>',
+  exportpdf_converter_options: {
+   	header_html: '<p>HEADER</p>', // example
+   	header_and_footer_css: 'p { color: blue }', // example
```

Comment:
As mentioned before there are other options that can be used. All the options with examples are listed [here](https://exportpdf.converter.tiny.cloud/docs#section/PDF-options)

Reply:
I think we can leave this as a example, now that we have added links to all the other options available.